task17/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>JS課題17</title>
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <script type="text/javascript" src="index.js" defer></script>
+  </head>
+  <body>
+    <div id="js-slideShowContainer" class="slideshow-container">
+      <div id="js-slideShowItem" class="slideshow-item">
+        <button id="js-prevBtn" class="prev-btn, slide-btn">
```

Comment:
IMO
prev-btn seems to be a class name that you no longer use, so we recommend that you remove it.

Reply:
Thanks for pointing that out! You are correct. I have corrected it.
https://github.com/seeeen044/JavaScript_handson/commit/b40f5885a0c9b028a5a8c81238f1d7bd0448f741